jupyter-js-widgets/examples/web3/src/index.js

```diff
@@ -29,7 +29,7 @@ document.addEventListener("DOMContentLoaded", function(event) {
         var inputarea = document.getElementsByClassName("inputarea")[0];
         var editor = CodeMirror(inputarea, {
             value: code,
-            mode: "python",
+            mode: 'python',
```

Comment:
Same here, the web3 example uses babel, hence we can probably revert the changes on index.js